py3status/modules/do_not_disturb.py

```diff
@@ -1,109 +1,225 @@
 # -*- coding: utf-8 -*-
 """
-Turn on and off dbus notifications.
+Turn on and off desktop notifications.
 
-A left mouse click will toggle the state of this module.
+This module supports dunst, xfce4-notifyd, and lxqt-notificationd.
+For best results, don't install more than one notification server.
 
 Configuration parameters:
-    dunst_signal_off: Signal to send to the dunst process to disable "Do Not Disturb."
-        (default 'SIGTERM')
-    dunst_signal_on: Signal to send to the dunst process to enable "Do Not Disturb."
-        (default 'SIGUSR1')
-    format: Display format for the "Do Not Disturb" module.
-        (default '{state}')
-    notification_manager: The process name of your notification manager.
-        (default 'dunst')
-    refresh_interval: Refresh interval to use for killing notification manager process.
-        (default 0.25)
-    state_off: Message when the "Do Not Disturb" mode is disabled.
-        (default 'OFF')
-    state_on: Message when the "Do Not Disturb" mode is enabled.
-        (default 'ON')
+    cache_timeout: refresh interval for this module; for xfce4-notifyd
+        (default 10)
+    format: display format for this module
+        (default '{server} [\?if=state&color=good DND|\?color=bad DND]')
+    pause: specify whether to pause or kill process; for dunst
+        (default False)
+    server: specify notification server to use, otherwise auto
```

Comment:
This needs the options specifying

Reply:
```
    server: specify notification server to use, eg dunst, xfce4-notifyd,
        or lxqt-notificationd, otherwise auto
        (default None)
```